crates/uv-cli/src/lib.rs

```diff
@@ -2000,11 +2085,21 @@ pub struct RunArgs {
     pub refresh: RefreshArgs,
 
     /// Run the command in a specific package in the workspace.
+    ///
+    /// If not in a workspace, or if the workspace member does not exist, uv
+    /// will exit with an error.
     #[arg(long)]
     pub package: Option<PackageName>,
 
-    /// Avoid discovering the project or workspace in the current directory or any parent directory.
-    /// Instead, run in an isolated, ephemeral environment populated by the `--with` requirements.
+    /// Avoid discovering the project or workspace.
+    ///
+    /// Instead of searching for projects in the current directory and parent
+    /// directories, run in an isolated, ephemeral environment populated by the
+    /// `--with` requirements.
+    ///
+    /// If a virtual environment is active or found in a current or parent
+    /// directory, it will be ignored and a new isolated environment will be
+    /// used.
```

Comment:
Ok, I changed this in https://github.com/astral-sh/uv/pull/5846. I think you can just strike this paragraph... As of that PR, we now _will_ reuse a virtualenv if it exists, just following our standard semantics, same as if there were no project...

Reply:
`--no-project --isolated` would instead use an ephemeral env.